mne/channels/channels.py

```diff
@@ -336,6 +339,78 @@ def rename_channels(info, mapping):
     # it doesn't work)
     info['ch_names'] = [c['ch_name'] for c in chs]
 
+    # assert channel names are unique
+    assert len(info['ch_names']) == np.unique(info['ch_names']).shape[0]
+
+
+def set_channels_type(info, mapping):
```

Comment:
just thought about it and i think it should. this should be handled by the user using the readers. do we add private functions to the whats_new?